extensions/Dockerfile

```diff
@@ -24,11 +38,19 @@ RUN createrepo_c /usr/share/rpm-ostree/extensions/
 # Generate extensions.json for meta.json.
 # Use dnf repoquery to print 'name: version,' for each RPM
 # sed to remove the comma from the last RPM
-RUN (echo "{" && \
-(dnf repoquery --repofrompath=extensions,/usr/share/rpm-ostree/extensions/ \
+RUN <<EOF
+set -xeuo pipefail
+
+# buildah doesn't seem to support heredoc output
+# redirection like buildkit so do it manually here
```

Comment:
is there a bug for this we can link to?

Reply:
I didn't find one, but I could file one I guess.

Ahhh OK, just saw your other comment below related to this. The feature I'm talking about here isn't just bash redirection, but a `RUN` feature so you can write e.g.

```
RUN <<EOF > /out
echo foobar
```

and it'll go to `/out`. Which would've been _perfect_ for our use case here.